### PR TITLE
Updating to the new versions OCPP1.6 schemas

### LINKED ISSUE #57: Implement errata v4 for OCPP 1.6
In the OCPP 1.6 schemas for MeterValues.req, OCA misspelled Celsius as Celcius.

In the OCPP  1.6 erratas, the recommended action to handle this is:

* For ChargingStations: send Celsius (ie the correct spelling), Celcius is also allowed.
* For CentralSystems: accept both Celsius and Celcius.

There are new schema files available from OCA.

### PR DESCRIPTION
**Pull request by darander**
OCA has released new versions of the OCPP1.6 schemas together
with erratas.

This is the latest version.

Fixes #57 (adding Celsius with correct spelling) and more.

Signed-off-by: Anders Darander <anders@chargestorm.se>

### CODE REVIEW

**Comment by darander** [comment]
Thanks for the comment.

I've updated the commit to omit the id lines.

File: ocpp/v16/schemas/StopTransaction.json [hunk 1]
```diff
@@ -1,166 +1,175 @@
...
+                                        "W",
+                                        "kW",
+                                        "VA",
+                                        "kVA",
+                                        "var",
+                                        "kvar",
+                                        "A",
+                                        "V",
+                                        "K",
+                                        "Celcius",
```
Comment (OrangeTux) [inline ocpp/v16/schemas/StopTransaction.json]: It seems that `Celsius` is missing here.

Comment (darander) [reply ocpp/v16/schemas/StopTransaction.json]: That's right.
I'll bring it back to OCA, as I prefer to not touch the schemas here. (I think it's better to use the upstream schemas if possible).

Comment (OrangeTux) [reply ocpp/v16/schemas/StopTransaction.json]: Fair point.
I guess that the OCA fix the bug in a new V5 errate. We can implement the fix than after the errata has been released.

For now I am ready to merge the PR. Thanks for your help!

**Comment by OrangeTux** [comment]
For some reason the builds on CirceCi haven't been run. I will try to fix that. If tests pass I'm happy to merge!

**Comment by darander** [comment]
I've added Celsius to StopTransaction.json, OCA is notified, but I rather not wait for them anyway.

I've also updated the commit message with info on when the new schema files were released and which modifications we've done to them.

File: ocpp/v16/schemas/StopTransaction.json (hunk 1 above)
Comment (tropxy) [reply ocpp/v16/schemas/StopTransaction.json]: I also have contacted the OCA. It seems that they on purpose want the schemas to remain unchanged and that, for now, CS must adapt the enumerations and include both spellings and CP must correct the spelling.

**Comment by OrangeTux** [comment]
@darander I fixed CircleCI's settings and now the test suite has ran. Unfortunately the build fails, can you fix it?

**Comment by darander** [comment]
Fixed the build.

Thanks for getting the CircleCI working again!

**Comment by OrangeTux** [comment]
Many thanks for your help!

**Comment by darander** [comment]
Thanks for your help with my contribution!

**Pull request merged**